spec/services/idv/profile_maker_spec.rb

```diff
@@ -45,8 +45,13 @@
           deactivation_reason: :encryption_error,
         )
 
-        expect(profile.active).to eq false
-        expect(profile.deactivation_reason).to eq 'encryption_error'
+        expect(profile.activated_at).to eq(nil)
+        expect(profile.active).to eq(false)
+        expect(profile.deactivation_reason).to eq('encryption_error')
+        expect(profile.fraud_review_pending?).to eq(false)
+        expect(profile.gpo_verification_pending_at.present?).to eq(false)
+        expect(profile.initiating_service_provider).to eq(initiating_service_provider)
```

Comment:
This is a nil == nil check. Maybe set the initiating SP to something other than nil? Or make this check against nil, rather than against `initiating_service_provider`, so that it's clear that `profile.initiating_service_provider` should be nil. (This comment applies in multiple places, but I won't clutter your PR by calling out all of them).

Reply:
checking for nil now, except when it is assigned.